src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Windows.cs

```diff
@@ -442,9 +442,21 @@ public SslProtocols SslProtocols
             _winHttpHandler.Properties :
             _socketsHttpHandler.Properties;
 
-        protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
+        protected internal override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
```

Comment:
I think the common case for `HttpClient` will be the default `TransferEncodingChunked == null`; I'd prefer if we didn't introduce all these extra allocations for async when they won't actually be used most of the time.

@stephentoub thoughts on this?

Reply:
> thoughts on this?

Yes, we should not do this.